### PR TITLE
[Add Local Addr & Port 2/3] Add columns to socket tracer

### PR DESCRIPTION
**Pull request by benkilimnik**
Summary: Adds data columns for local IP address and port to the socket tracer, which are populated by #1808 and #1809. This will support standalone pem entity relationships.

Type of change: /kind feature

Test Plan: Existing targets

### CODE REVIEW

File: src/stirling/source_connectors/socket_tracer/conn_tracker.h
Comment (etep) [inline src/stirling/source_connectors/socket_tracer/conn_tracker.h:810]: I think this was supposed to have the text `local_addr=$1:$2`.

Comment (benkilimnik) [reply src/stirling/source_connectors/socket_tracer/conn_tracker.h:810]: Fixed

**Review by etep — APPROVED** [review]

**Comment by oazizi000** [comment]
We've historically relied on only capturing the remote addr, and using the PID along with K8s metadata to determine the local IP, but I do see value in capturing the local IP directly. So this looks good, except that this can't be landed in isolation. Will be looking at the follow-up PRs soon.

**Review by JamesMBartlett — APPROVED** [review]
I think this will lead to empty strings in the relevant tables, but as long as we don't release before the other diffs in this series land I think its ok.

**Comment by JamesMBartlett** [comment]
Will hold off on landing until I have more confidence the other diffs will land.

**Comment by oazizi000** [comment]
> I think this will lead to empty strings in the relevant tables, but as long as we don't release before the other diffs in this series land I think its ok.

The other way to deal with this is to order things differently. It should be safe to do the data collection in bpf without actually pushing to the tables. And then have this diff come after that. Might require a bit of reshuffling, but logically that's how I would go about it (start from BPF code and layer diffs all the way up to pushing into the tables).

**Comment by benkilimnik** [comment]
Thanks for the feedback @oazizi000 @JamesMBartlett 
I've reordered the PRs such that data collection in bpf comes first #1808

**Pull request merged**
